src/server/common/helpers/fetch/fetch-entities.js

```diff
@@ -0,0 +1,16 @@
+import qs from 'qs'
```

Comment:
nit: more of a preference. I have started putting fetch methods in a `fetcher.js` file - much easier having them all in 1 place. Don;t worry for this work but for the future would be worth considering this